tests/integration/mounts_sshfs.bats

```diff
@@ -84,27 +88,25 @@ function setup_sshfs() {
 					options: ["dev", "suid", "exec", "atime", "rprivate", "rbind", "sync"]
 				}]'
 
-	runc run --no-mount-fallback test_busybox
-	# The above will fail as we added --no-mount-fallback which causes us not to
-	# try to remount a bind mount again after the first attempt failed on source
-	# filesystems that have nodev, noexec, nosuid, noatime set.
+	runc run test_busybox
+	# The above will fail because we explicitly requested a mount with several
+	# MNT_LOCKED mount options cleared (when the source mount has those mounts
+	# enabled).
 	[ "$status" -ne 0 ]
 	[[ "$output" == *"runc run failed: unable to start container process: error during container init: error mounting"*"operation not permitted"* ]]
 }
 
-@test "runc run [ro bind mount of a nodev,nosuid,noexec,noatime fuse sshfs mount without fallback]" {
-	setup_sshfs "nodev,nosuid,noexec,noatime"
+@test "runc run [ro,noexec bind mount of a nosuid,noatime fuse sshfs mount]" {
+	requires rootless
+
+	setup_sshfs "nosuid,noatime"
```

Comment:
Can you keep this test too?

The issue this PR fixed was a mount by containerd that only requested `"rbind", "ro"` while de source had `"nosuid", "noexec", "nodev"`.

See here: https://github.com/containerd/containerd/pull/8309/files

Reply:
Sure, though the reason for changing it is that a proper test of the flag functionality involves explicitly disabling a mnt_locked flag with a mount that has other mnt_locked flags not specified. But I can keep this one as well (and add a reference to containerd/containerd#8309).